configure

```diff
@@ -4323,7 +4325,7 @@ else $as_nop
 
 fi
 
-# Check whether --enable-host_gcc was given.
+# Check whether --enable-host-gcc was given.
```

Comment:
Is this correct?

Reply:
Does not matter much. That's a generated comment in a generated script.
And since this is generated, we should not edit the generated code/comments.